### PR TITLE
Verify if stress commands have all files to be executed

### PR DESCRIPTION
**Pull request by dkropachev**
https://trello.com/c/iHuKBX0F/1565-verify-that-userprofile-file-exist-before-the-tests-starts-and-resources-provisioned

## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I gave variables/functions meaningful self-explanatory names
- [x] I didn't leave commented-out/debugging code
- [x] I didn't copy-paste code
- [x] I added the relevant `backport` labels
- [ ] ~~New configuration option are added and documented (in `sdcm/sct_config.py`)~~
- [x] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [x] All new and existing unit tests passed (CI)
- [ ] ~~I have updated the Readme/doc folder accordingly (if needed)~~

### CODE REVIEW

**Comment by bentsi** [comment]
1) pylint failed
2) rebase on master and follow the "Commit message format in SCT" email

**Comment by dkropachev** [comment]
> 1. pylint failed
> 2. rebase on master and follow the "Commit message format in SCT" email

Done

File: sdcm/tester.py [hunk 1]
```diff
@@ -139,6 +139,7 @@ def __init__(self, *args):  # pylint: disable=too-many-statements
         self.status = "RUNNING"
         self.params = SCTConfiguration()
         self.params.verify_configuration()
+        self.params.check_required_files()
```
Comment (fruch) [inline sdcm/tester.py]: Can also be added to one of the unit tests of sct_events ?

**Comment by dkropachev** [comment]
> please add a unit test for the new code in `test_config.py`

Added

**Pull request merged**